### ISSUE TITLE
Create Social Assets for Initial Fundraising Tests

### ISSUE THREAD

**Issue by HPaulJohnson** [MozillaFoundation/Advocacy#1057]
Detailed description of work: Create a set of 5 social posts (Facebook and Twitter) to allow us to begin testing our fundraising ask on social and to start to build a audience segment of potential donors on these platforms

*Start date: September 18, 2016* 
*Due date: October 2, 2016* 
Note: The dates need to be exactly in the format indicated above for proper integration with the Gantt chart.

CC:
@beccaklam @tlasadea

**Comment by hannahkane** [comment]
@HPaulJohnson @tlasadea - can we provide guidance about messaging or look&feel?

**Comment by HPaulJohnson** [comment]
@hannahkane yes!


I'd suggest we pull from two things: what worked best last year (I can dig that up) and what has worked in other channels so far this year, i.e. what snippet message has performed best in early testing.

**Comment by HPaulJohnson** [comment]
Here's some background on last year + snippet:

Social Plan & Assets from last year: 
https://docs.google.com/document/d/1Pg-4PWXu8z5y0T5Il6VYxZU__T5QwucoqiBX9jV5xPc/edit

I'm having trouble finding the report from our test but I do remember that the Photo version was the winner by a significant margin. 

Here's the ticket where we tracked the various designs for EOY social: https://github.com/MozillaFoundation/Advocacy/issues/606

Finally, here are the results from the snippet so far this year:
https://docs.google.com/spreadsheets/d/1SKw3z7IFEpHkaXfj5RKE7Ypcp1NWsg-tRKf73Mk_WVw/edit#gid=1962598012

**Comment by hannahkane** [comment]
@HPaulJohnson @natalieworth - were you guys able to discuss? Can you summarize what you agreed in terms of deliverables/deadlines?

**Comment by natalieworth** [comment]
@hannahkane Paul's working on getting me copy for this, but in the meantime I'm starting some directions. Paul suggested starting 3 directions now (based on the testing from last year from what was successful) using internet health, and possibly the net neutrality asset style that Jordan created or the internet video that we made and tweaking the copy at the end. Things to consider were keeping a low amount of text on the assets and that photography was winning last year on facebook. Themes could be based off of the snippet content tests.

**Comment by natalieworth** [comment]
Have rough thoughts + drafts for social here: https://redpen.io/p/gmcf01c9fa3d98de75
cc @HPaulJohnson @tlasadea @beccaklam @hannahkane 
Went with the themes from the snippet copy above (focusing on internet health, women + web literacy, and our advocacy/ net neutrality work) with additional options.

**Comment by HPaulJohnson** [comment]
Thanks, @natalieworth - lots of great stuff to work with. In an effort to refine the list here's what I'm suggesting we test/itereate on in a 2nd round.

**Test 1: Photography**
This is essentially the winner from testing in 2016 so let's start with this. I'd like to see two options, 1 with type/CTA and one without (last year, without any type or treatment was the winner). 

I think designs 9 & 10 are closest. 

**Test 2: Neon** 
The neon designs feel similar to the aesthetic of much of other work this year and, assuming those are the same people who are likely to give us money, I like the idea of having some tests that have a similar feel. 

I think designs 13 & 15 work best for this. 

**Test 3: Heart**
We've also used the heart extensively in our fundraising and like the idea of testing it as well. We could test static vs. .gif within this treatment as well. 

Any of the designs 19-21 work well for this. 

**Test 4: Bold type on bright background**
I think these really standout though the copy needs some work to put some distance between us and firefox. I think we can take some of the winning snippet copy, shorten/tighten it and apply it to this social format. 

Design 17 is the one I'd like to riff off. 

**Test 5: Advocacy-specific message**
Assuming our active advocates are also our likely donors and knowing that referencing recent/active advocacy campaigns worked in snippet, I'd like to run an advocacy-specific test where, instead of "sign this petition" as a CTA we swap in "donate to Mozilla." We could run one of your new designs against the winning image from our Net Neutrality campaign ("Save the Internet") as a solid test here. 

I think I like design 26 the best for the test, though the collage idea could work given some tweaking.

**Comment by HPaulJohnson** [comment]
Next steps for me: put this into a test matrix and start a copy deck. 

@natalieworth let me know if you want to discuss any of the above and/or if you have enough to be unblocked for now.

**Comment by natalieworth** [comment]
Thanks @HPaulJohnson @tlasadea @beccaklam for the feedback! I've updated the social redpen here https://redpen.io/p/gmcf01c9fa3d98de75 with the second round of iterations! Thanks

**Comment by HPaulJohnson** [comment]
These are looking great and I think really close to being done. 

I wrote some copy for each and included what I think would be the final designs we'd want to test with here: https://docs.google.com/document/d/1lNeyBFUijTKqEjzIQIPFcqwpvtJA1QRQUHVBuicZvjE/edit

I'd love if @tlasadea and @WillatMozFdn could look at the copy in particular and provide any feedback or edits. 

RE: final design tweaks the only changes I might have are to the "bold type" versions. I'd like to think about a few more copy approaches there and will work on that shortly. Otherwise, I'm good with these images for this initial round of posting/testing. 

Awesome work.

**Comment by tlasadea** [comment]
@hpauljohnson added comments. can discuss on our call shortly if you like.

**Comment by WillatMozFdn** [comment]
Overall, I love these!  Thanks!  Added a few suggestions.  talk to you guys in a bit here.

**Comment by natalieworth** [comment]
Added the social folder here (with the exception of the bold type option, just waiting on finalized copy for those): https://drive.google.com/drive/folders/0B20-0lpP23c0RUJKODl3ekN2MWs?usp=sharing
cc @HPaulJohnson @tlasadea @WillatMozFdn

**Comment by hannahkane** [comment]
Are we good to close this ticket, @HPaulJohnson @WillatMozFdn @natalieworth ?

**Comment by HPaulJohnson** [comment]
After pausing for a brief while to get the overall messaging brief for EOY approved, let's restart this ticket for an initial social test we are aiming to run between 11/6-11/15 (pre-Quantam launch). 

We will use the above assets created by @natalieworth as a control against a new set of tests being developed by myself, @tlasadea and @kristinashu.

This new test will include 6 new social assets that hit upon a more topical/urgent tone. We've narrowed down topics for these test assets to: 

- Digital Inclusion with Coral Project as a proof point
- Misinformation with Fellow(s) as a proof point
- Decentralization with Net Neutrality advocacy as proof point. 

Copy and initial designs being developed now and feedback and iteration to happen next week in London, pre-Mozfest. Final polishing to be done week following, ideally by 11/1.

**Comment by HPaulJohnson** [comment]
@kristinashu moodboards to help inform design direction for tests: https://docs.google.com/presentation/d/11BPDSJPX2denC9vzPjCuRw1wcWNLZKdonopa2IFd11c/edit#slide=id.g2812d1188b_0_29

**Comment by HPaulJohnson** [comment]
also, here's @tlasadea's brief for the test messages:

https://docs.google.com/document/d/1SrEAlwmCLG4c3zk15uyb2MW2gMA9pK5CNg0PAa_oXbk/edit

**Comment by kristinashu** [comment]
@HPaulJohnson @tlasadea here is a very rough wip https://redpen.io/p/zj3a03844f39d2841c I haven't touched any of the copy but will update when we have something. I'll continue to add more directions but please add any thoughts/comments/ideas you have.

**Comment by tlasadea** [comment]
@kristinashu @HPaulJohnson copy for these ads are here: https://docs.google.com/spreadsheets/d/18_9IPhiB8SC9mtViBvkLCI1kV8LTJbdJzfLHFpITEPk/edit#gid=962456791

**Comment by natalieworth** [comment]
Round 1 social updates here: https://redpen.io/p/bseb9ca7649f8f90f4
cc @HPaulJohnson @tlasadea @beccaklam

**Comment by natalieworth** [comment]
Round 1 social updates here: https://redpen.io/p/bseb9ca7649f8f90f4
cc @HPaulJohnson @tlasadea @bec

**Comment by tlasadea** [comment]
Whoops @natalieworth this is the wrong ticket for the above red pen, this is for the EOY Fundraising, not buyer's guide social. Will move to correct ticket.

**Comment by tlasadea** [comment]
Hey @kristinashu - copy for creative (misinformation headlines + decentralization posters) are [here](https://docs.google.com/spreadsheets/d/18_9IPhiB8SC9mtViBvkLCI1kV8LTJbdJzfLHFpITEPk/edit#gid=1643780827), tab number 4!

**Comment by kristinashu** [comment]
Thanks for the copy and feedback. Here are the updated images https://redpen.io/p/zj3a03844f39d2841c 

Let me know if you have any more feedback or have anymore ideas.

**Comment by HPaulJohnson** [comment]
thanks, @kristinashu 

@tlasadea is on PTO today but i'll let her have the final say on Monday a.m. If she doesn't have any additional feedback we can export them and get the campaign up and running.

**Comment by kristinashu** [comment]
All the images and working files are here: 
https://drive.google.com/open?id=1DaZmbqY0_dwNhDPItTWdDIC4nHxf9hz-

**Comment by HPaulJohnson** [comment]
This is done/live. Will create a new ticket for phase 2 once we look at the results.
